### DISCUSSION TITLE
Creando una proforma con SDK php e traformandola in fattura da portale è possibile trameite id proforma trovare id fattura sempre SDK?

### DISCUSSION THREAD

**Discussion by thebetti** [fattureincloud/api#220] [Q&A] [answered]
Buongiorno,
Creando una proforma con SDK php e traformandola in fattura da portale è possibile trameite id proforma trovare id fattura generato ( ovviamente tramite SDK PHP)  con tutte le informazioni aggiornate? ho trovato tramite api reference  i campi merged_in le informazion ima nell'SDK questi campi non sono presenti.

**Comment by emajo** [comment, marked as answer]
Ciao @thebetti il campo merged_in non è documentato e non è presente nelle nostre SDKs perché verrà deprecato, il mio consiglio è di salvare lato vostro queste informazioni.

Grazie,
Emanuele.

**Reply by thebetti** [reply]
Si salve ma quali informazioni devo salvare ? dalla proforma come faccio a risalire alla fattura?

**Reply by emajo** [reply]
Sarebbe sufficiente andare a salvare gli id del documento padre (in questo caso la proforma) e del documento figlio (fattura).

**Reply by thebetti** [reply]
ok ma come ricerco il documento FATTURA , dato il ID PROFORMA?

**Reply by emajo** [reply]
Ipotizzando di salvare i due dati su una tabella 
| parent_document | child_document |
|-----------------|----------------|
| 12345           | 67890          |

avendo l'id proforma (12345) può essere recuperato l'id della fattura con una query del tipo 
`SELECT child_document WHERE parent_document = 12345`
otteniamo così l'id 67890 che può essere passato in una chiamata [get issued document](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents/-document_id-) per ottenere i dati della fattura.

**Reply by thebetti** [reply]
@emajo Ciao Grazie mille per la tua disponibilità, ma forse mi sono spiegato male ci riprovo.
1) dal portale web  un utente ( es utente1 ) tramite SDK Genera una proforma sulla vostra piattaforma ID 1234 (che il programma memorizza in una tabella ID PROFORMA  da voi trasmesso, come responce dal servizio Create Issued Document.
2) un'altro utente ( utente 2), accede al vostro portale web si trova la proforma creata al punto uno , la visiona la modifica ecc ecc e la trasforma in fattura  (con nuovo ID 4567).
3) utente1  vuole vedere se La PROFORMA (ID 1234) da lui creata è diventata una fattura. Come posso tramite SDK PHP avendo Id PROFORMA (ID 1234 ) capire se il documento si è trasformato in fattura o in qualsiasi altro tipo di documento? (ovvere ricercare ID fattura 4567)
grazie mille

**Reply by thebetti** [reply]
@emajo sono riuscito a spiegarmi?

**Reply by emajo** [reply]
Ciao @thebetti si, ho capito la problematica, sto aspettando conferma ma posso dirti con quasi certezza che attualmente non c'è modo di recuperare queste informazioni

**Reply by thebetti** [reply]
@emajo grazie mille per la disponibilità attendo sperando che il principio di ereditarietà sia mantenuto
